blockchain/message_pool/src/msgpool/mod.rs

```diff
@@ -339,11 +339,11 @@ pub mod tests {
     #[cfg(feature = "slow_tests")]
     use std::borrow::BorrowMut;
     #[cfg(feature = "slow_tests")]
-    use std::thread::sleep;
-    #[cfg(feature = "slow_tests")]
     use std::time::Duration;
     use test_provider::*;
     use tokio::task::JoinSet;
+    #[cfg(feature = "slow_tests")]
+    use tokio::time::sleep;
```

Comment:
This import is unused. Could you add `cargo check --tests --features slow_tests` to the `test` target in the makefile?